components/stats/LoggedDatesCalendar.tsx

```diff
@@ -24,7 +25,7 @@ const LoggedDatesCalendar = () => {
     }
   );
 
-  if (isLoading) return null;
+  if (isLoading) return <CardSkeleton count={1} height={height} width={width} />;
```

Comment:
How would it be if this was isFetching like we did in other components?

Reply:
I knowingly did it that way, because I don't want the loading screen to show up each time